app/controllers/results_controller.rb

```diff
@@ -0,0 +1,12 @@
+class ResultsController < ApplicationController
+  def index
+    @course = Course
```

Comment:
Should this be `@courses`? Also, could this go in a query object or a scope on `Course`. This controller only really passes in pagination params but it needs to know a lot of relationships if we build the query in here.

Reply:
Since we're requesting this data over the wire (via JSON:API) it's better that we explicitly include relationships that we want.

Reply:
Ah right. Ok 👍